bo4e/models/bo/Bilanzierung.json

```diff
@@ -2,20 +2,20 @@
   "additionalProperties": true,
   "description": "Bilanzierung is a business object used for balancing. This object is no BO4E standard and a complete go\nimplementation can be found at\nhttps://github.com/Hochfrequenz/go-bo4e/blob/3414a1eac741542628df796d6beb43eaa27b0b3e/bo/bilanzierung.go",
   "properties": {
-    "versionstruktur": {
+    "_version": {
       "default": "2",
-      "title": "Versionstruktur",
+      "title": " Version",
       "type": "string"
     },
-    "boTyp": {
+    "_typ": {
       "allOf": [
         {
           "$ref": "../enum/Typ.json#"
         }
       ],
-      "default": "BILANZIERUNG"
+      "default": "Bilanzierung"
```

Comment:
Warum ist das eigentlich kein Uppercase mehr? Haben wir da was in den Enum-values von `Typ` verkackt bzw. ist das inkonsistent?

Reply:
Mit der Uppercase-Variante habe ich Probleme beim type_check bekommen. Wahrscheinlich, weil in der typ.py '"BILANZIERUNG = "Bilanzierung"' steht und er daher "BILANZIERUNG" als String und nicht als Typ ansieht.